companies.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+$suppLang = ['sv', 'fi', 'en'];
+// XXX: PLEASE SET THESE ACCORDINGLY BEFORE GOING INTO PRODUCTION
+$langArticleMap = [
+    'sv' => '614',
+    'fi' => '613',
+    'en' => '612'
+];
+
+$lang = $suppLang[0];
+if (isset($_GET['lang']) && in_array((string)$_GET['lang'], $suppLang))
+  $lang = $_GET['lang'];
+
+$forCompaniesObj = [
+    'proto' => 'https',
+    'domain' => 'about.teknologforeningen.fi',
+    'path' => '/index.php',
+    'getParam' => [
+        ['option', 'com_content'],
+        ['view', 'article'],
+        ['id', $langArticleMap[$lang]]
+    ]
+];
+
+$location = $forCompaniesObj['proto'].'://'.$forCompaniesObj['domain'].$forCompaniesObj['path'];
+$nGetParam = count($forCompaniesObj['getParam']);
+
+if($nGetParam > 0) {
+    $location .= '?';
+    for($getParamIdx = 0; $getParamIdx < $nGetParam; $getParamIdx++) {
```

Comment:
for heavens sake, use `foreach`, and why not just use http://php.net/manual/fa/function.http-build-url.php

Reply:
Good point about the built-in `http-build-url`. Foreach is very nice indeed, but would require an additional variable to track how n:th element we are on, which we use in the conditional statement about adding an ampersand to the location string in case we are NOT at the last parameter. But sure, will use `http-build-url` before resubmitting (this will also get rid of my glorious loops).

Reply:
http-build-url isn't present in atleast the docker dev server env, but the queryparams got fixed with `http_build_query()`, so that should resolve this.